### DISCUSSION TITLE
use a Make file for build process

### DISCUSSION THREAD

**Discussion by ceteri** [DerwenAI/pytextrank#258] [Ideas]
We need to move the execution of CLI scripts used to build the documentation into a Make file.  Wise suggestion from @louisguitton: <https://github.com/DerwenAI/pytextrank/pull/159#issuecomment-832492929>

**Comment by louisguitton** [comment]
the goal would be to duplicate what is documented in https://derwen.ai/docs/ptr/build in the makefile as a form of extra documentation
